app/browser/api/ledgerNotifications.js

```diff
@@ -199,11 +224,38 @@ const showEnabledNotifications = (state) => {
     } else if (shouldShowNotificationAddFunds()) {
       showAddFunds()
     }
-  } else if (reconcileStamp - new Date().getTime() < 2 * ledgerUtil.milliseconds.day) {
+  } else if (reconcileStamp - now < 2 * ledgerUtil.milliseconds.day) {
     if (sufficientBalanceToReconcile(state) && (shouldShowNotificationReviewPublishers())) {
-      showReviewPublishers(new Date().getTime() + ledgerUtil.milliseconds.day)
+      showReviewPublishers(now + ledgerUtil.milliseconds.day)
+    }
+  } else if (hasFunded(state) && !aboutPreferencesState.hasBeenBackedUp(state)) {
+    const backupNotifyCount = aboutPreferencesState.getPreferencesProp(state, 'backupNotifyCount') || 0
+    const backupNotifyTimestamp = aboutPreferencesState.getPreferencesProp(state, 'backupNotifyTimestamp') || backupNotifyInterval[0]
+    if (!bootStamp) {
+      bootStamp = now
+      state = ledgerState.setInfoProp(state, 'bootStamp', bootStamp)
+    }
+    if (now - bootStamp > backupNotifyTimestamp) {
+      const nextTime = backupNotifyTimestamp + getNextBackupNotification(state, backupNotifyCount + 1, backupNotifyInterval) // set next time to notify case for remind later
+      state = aboutPreferencesState.setPreferencesProp(state, 'backupNotifyCount', (backupNotifyCount + 1))
+      state = aboutPreferencesState.setPreferencesProp(state, 'backupNotifyTimestamp', nextTime)
+      module.exports.showBackupKeys(nextTime)
+    }
+    if (process.env.LEDGER_NOTIFICATION) {
```

Comment:
`process.env.LEDGER_NOTIFICATION` -> `process.env.LEDGER_VERBOSE`

Reply:
Are you sure you want to put this under verbose? It would be nice if QA saw just the time passed rather than filter through all the other messages. Also, when run in the future, you'll get the time thrown out to the console every 5 seconds.

Reply:
that's a good point. Let's keep it like this then